Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -175,9 +178,8 @@
   - type: EnergySword
     litDamageBonus:
         types:
-            Slash: 9
-            Heat: 9
-            Structural: 20
+            Slash: 17
+            Heat: 17
```

Comment:
```suggestion
            Slash: 10
            Heat: 10
            Structural: 20
```

Reply:
If we're going for a buff, don't give it a huge nerf. D Sword was also made to be a breaching tool & speed vs the Energy Sword which is made to be an attack tool.

D Sword should **not** be "Esword but better"

Reply:
Who actually uses the D esword as a breaching tool? Nukies can't buy them while traitors usually just go with the fireaxe, which is better than the D esword in most scenerios. Currently melee meta is to attack your enemy, then pull back till recharge is over, making the speed obsolete unless you are extremely caught off guard.

Reply:
I'm going with more of a ss13 desword vibe, where it is the esword but double (3 hits to kill unarmored instead of 4, 100% reflection for energy guns) and no one actually uses the desword at all anymore due to it being able to be countered by 2 bolas being thrown in their general direction or stun baton

Reply:
> Who actually uses the D esword as a breaching tool? Nukies can't buy them while traitors usually just go with the fireaxe, which is better than the D esword in most scenerios. Currently melee meta is to attack your enemy, then pull back till recharge is over, making the speed obsolete unless you are extremely caught off guard.

Nukies should be able to buy them, unless it was a huge issue.
D Swords should also be attacking faster than a regular esword.

D Sword is also suppose to be used as a engage/disengage tool (reflect) and alpha strike (speed to catch off guard)

Reply:
Also, why does the **double** bladed energy sword deal less damage than the **one bladed** energy sword.

Reply:
Because it doesn't need to be energy sword but better. There's no arguing it.

Reply:
imo would be better for breaching if you could cut open doors like the opening of star wars episode 1, the phantom menace. without it theres very little justification to buy it (for almost all your tc) as a traitor and its blacklisted from nukies

Reply:
This suggestion also fundamentally reverses my PR

Reply:
No, I am not going to commit this suggestion.

Reply:
Here's the deal. If the damage gets buffed insanely, the TC cost must go to 20 and disabled from discounts.